coverage/test/env_test.go

```diff
@@ -18,8 +18,12 @@ package test
 
 import (
 	"testing"
+	"path"
 )
 
 func Test(t *testing.T) {
-	t.Logf(absPath("middle/of/nowhere"))
+	pathRelToProj := "middle/of/nowhere"
+	expected := path.Join(ProjDir(), pathRelToProj)
```

Comment:
You're basing both the expected and actual values on the code being tests. Hardcode the project dir here, otherwise the test may pass even if `ProjDir()` has an error.

Reply:
actually I cannot hardcode the project dir here because it depends either on GOPATH or an env var TEST_SRCDIR. I updated this part, please see if you are ok with the update.

Reply:
You're overriding any value set in line 30, which renders lines 28-31 useless. I suspect there's an else misssing here?